packages/react-reconciler/src/__tests__/ReactIncrementalErrorReplay-test.internal.js

```diff
@@ -10,21 +10,22 @@
 'use strict';
 
 describe('ReactIncrementalErrorReplay-test', () => {
+  const React = require('react');
+  const ReactTestRenderer = require('react-test-renderer');
```

Comment:
This test didn't load anymore without a real renderer because internal module transitive deps now reached the host config.